### DISCUSSION TITLE
Flutter builds timing out on Installing SDKs step

### DISCUSSION THREAD

**Discussion by warpaltarpers** [codemagic-ci-cd/codemagic-docs#2436] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Start Flutter build from `codemagic.yaml` set-up

### Expected results

Build completes and distributes during post-processing.

### Actual results

Build fails during **Installing SDKs** step, timing out when installing Flutter from stable channel.
```
== Use Flutter channel stable ==
> curl --fail --show-error --silent --output flutter_macos_arm64_3.19.4-stable.zip --connect-timeout 10.00 --max-time 120.00 https://storage.googleapis.com/flutter_infra_release/releases/stable/macos/flutter_macos_arm64_3.19.4-stable.zip
curl: (28) Operation timed out after 120009 milliseconds with 641469680 out of 1168340724 bytes received

Downloading Flutter release timed out

Build failed :|
Downloading Flutter release timed out
```

### Build id (optional)

66019f046609d8ffec0db3db

**Comment by warpaltarpers** [comment]
Just saw [the incident on the status page](https://status.codemagic.io/incident/345831), can close if being handled elsewhere.

**Comment by mikhail-tokarev** [comment]
please take a look on status.codemagic.io 
and this thread https://github.com/orgs/codemagic-ci-cd/discussions/2424
